src/components/NumericStepper/__tests__/NumericStepper.spec.tsx

```diff
@@ -0,0 +1,278 @@
+        expect(stepperButtons[0]).toHaveAttribute('disabled');
+    });
+
+    it('should set bottom edge value if user enters value below allowed limit', async () => {
+        const user = userEvent.setup();
+        view = render(<NumericStepper onChange={onChangeMock} minValue={10} />);
+        const inputButton = screen.getByRole('spinbutton');
+
+        await user.type(inputButton, '9');
+
+        expect(inputButton).toHaveAttribute('value', '10');
+    });
+
+    it('should set top edge value if user enters value above allowed limit', async () => {
+        const user = userEvent.setup();
+        view = render(<NumericStepper onChange={onChangeMock} maxValue={10} />);
+        const inputButton = screen.getByRole('spinbutton');
+
+        await user.type(inputButton, '11');
+        await inputButton.blur();
+
+        expect(inputButton).toHaveAttribute('value', '10');
+    });
+
+    it('should set value to the previous one when user enters invalid value', async () => {
+        const user = userEvent.setup();
+        view = render(
+            <NumericStepper onChange={onChangeMock} minValue={2} maxValue={4} defaultValue={3} />
+        );
+
+        await increaseClick();
+
+        expect(onChangeMock).toBeCalledTimes(1);
+        expect(onChangeMock).toBeCalledWith(4);
+        expect(screen.getByRole('spinbutton')).toHaveAttribute('value', '4');
+        expect(onChangeMock).toHaveBeenCalledWith(4);
+
+        const inputButton = screen.getByRole('spinbutton');
+        // Imagine users enters empty string
+        await user.type(inputButton, '   ');
+
+        inputButton.blur();
+
+        expect(inputButton).toHaveAttribute('value', '4');
+    });
+
+    it('should correctly react on increase click when top edge is overstepped', async () => {
+        view = render(
+            <NumericStepper onChange={onChangeMock} step={2} maxValue={4} defaultValue={3} />
+        );
+
+        await increaseClick();
+
+        expect(onChangeMock).toBeCalledWith(4);
+        expect(screen.getByRole('spinbutton')).toHaveAttribute('value', '4');
+        expect(onChangeMock).toHaveBeenCalledWith(4);
+    });
+
+    it('should correctly react on decrease click when bottom edge is overstepped', async () => {
+        view = render(
+            <NumericStepper onChange={onChangeMock} step={2} minValue={3} defaultValue={4} />
+        );
+
+        await decreaseClick();
+
+        expect(onChangeMock).toBeCalledWith(3);
+        expect(screen.getByRole('spinbutton')).toHaveAttribute('value', '3');
+        expect(onChangeMock).toHaveBeenCalledWith(3);
+    });
+
+    // it('should react on keyup/keydown press', () => {
+    //     /**
+    //      * TODO: Test with E2E tests.
+    //      * As we actually validate data on blur, its quite challenging to chain together mouseEnter, keyDown and blur.
+    //      * I've tried multiple ways to do it, and don't want to spend way more time on that.
+    //      */
+    // });
```

Comment:
This comment is about Enzyme. Now it should be easy. Can you please add a simple test to see that the buttons with with keyboard interaction? Just a click on each button, see that the value has changed... something simple like that.

Reply:
Do you mean this ? 
![image](https://github.com/textkernel/oneui/assets/104063210/407bcc5e-ddae-46c0-a7ab-1a71b889e0c6)

Reply:
This test is failed

Reply:
I've added